dev-python/urwidtrees/urwidtrees-1.0.2-r1.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+PYTHON_COMPAT=( python2_7 python3{5,6,7} )
```

Comment:
I wonder how you managed to get this to work.

Reply:
Ah yes was a mistake I should have seen that with ebuildtester sorry !